### ISSUE TITLE
How to combine with normal onPress ?

### ISSUE THREAD

**Issue by alfonsodev** [johanneslumpe/react-native-gesture-recognizers#6]
can this module respond  to normal onPress events ?

**Comment by johanneslumpe** [comment]
No, currently the decorators cannot handle that. If each decorator should also be able to handle normal `onPress` event, then their logic would have to be updated to decide whether it was just `onPress` or a swipe/pan. In case you only need `onPress`, you could easily write your own decorator for that.
